acceptance-tests.md

```diff
@@ -9,25 +9,32 @@ Reference for NERC OpenStack: [https://nerc-project.github.io/nerc-docs/get-star
 
 1. **Request a new account**
     * _As a new user, I should be able to create an account for myself in the NERC._
-    * <span style="text-decoration:underline;">Steps</span>
+    * Steps
         1. A prospective user follows the steps documented in [https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/](https://nerc-project.github.io/nerc-docs/get-started/create-a-user-portal-account/) to create an account on the NERC.
+    * Acceptance tests:
+        1. For a given username of `computate` for example, [Click here to check that the given username is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~User&name=computate). Note the identities for the given user, specifically one that starts with `github:`, like `github:15114160`.
+        2. For a given users identity, `github:15114160` for example, [Click here to check that the given identity is listed in the OpenShift API](https://console-openshift-console.apps.nerc-ocp-prod.rc.fas.harvard.edu/search/all-namespaces?kind=user.openshift.io~v1~Identity&name=github%3A15114160).
 2. **Remove a user**
     * _As an administrator, I should be able to remove a user, labels association with the user, and user identity from the cluster._
        * _(This step is generally unnecessary, given that a user that doesn’t have authorization to any project, cannot access any allocations, therefore removing a user from allocations is generally enough.)_
```

Comment:
Unclear which step you are saying is unnecessary.  The use case is to remove a user, and the user probably does have projects and allocations associated with their account.  All user information should be removed for the acceptance test to pass.

Reply:
@knikolla I see that you added these comments about this step being unnecessary in this "NERC/OpenShift Acceptance Tests" document: https://docs.google.com/document/d/18AUzxuFWKeNOO6UFgze6Tm1mF1Gz-UIMj8Wqu6lyxcc/edit
Would you be able to help us clarify this use case, "Remove a user"?

Reply:
Please revise the parenthetical text in this section and update the use case words quoted here in italics  to match the change in your other approved PR revising the use case for removing a user

Reply:
@computate ^